integration/render_test.go

```diff
@@ -688,7 +687,6 @@ kind: Pod
 metadata:
   labels:
     app.kubernetes.io/managed-by: SOMEDYNAMICVALUE
-    skaffold.dev/run-id: SOMEDYNAMICVALUE
```

Comment:
didn't catch this at first but looks like this is the wrong line being removed here right?

Reply:
Oof, thanks for catching that. Fixed.